tasks/main.yml

```diff
@@ -3,33 +3,34 @@
   template: src=ssh_config dest=/root/.ssh/config owner=root group=root mode=0600
 
 - name: check if puppet is already configured
-  command: "ls /etc/puppet/rack/public"
+  command: "ls {{puppet_etc_dir}}"
   register: puppet_configured
   ignore_errors: True
 
+- name: check if puppet.conf is OK
+  command: "grep -c 'server = ' {{puppet_etc_dir}}/puppet.conf"
```

Comment:
Why is this needed?

Reply:
The reason why this specific task "check if puppet.conf is OK" was added because when I first started this sprint to look at this the puppetmaster was in an unhappy state (directory there but puppet.conf did not point anywhere) but because the previous check found an existing directory a task below did not template in the puppet.conf

I think I would ask: Why are both of these "check if puppet is configured" needed.

Now. If nothing else seems to manage the puppet.conf on the puppetmaster I think we can actually remove the checks and just always manage these via templates..

Reply:
Turns out it's nice to have the old ls etc/dir check to determine if we want to create puppetmaster certificates or not. Adjusted a bit.